ofl/jaini/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,27 @@
+<p>
+    Jaini and Jaini Purva (both Devanagari) are typefaces based on the calligraphic style of the Jain Kalpasutra
+    manuscripts,
+    particularly referencing a manuscript from 1503 CE. The manuscript style has several unique features not seen in the
```

Comment:
@yanone, Same comment as in Jaini Purna: the formatting of the description file looks broken, please adjust it following this https://github.com/fonttools/fontbakery/issues/2811